ga4gh/datamodel/sequenceAnnotations.py

```diff
@@ -158,14 +158,26 @@ class AbstractFeatureSet(datamodel.DatamodelObject):
     """
     compoundIdClass = datamodel.FeatureSetCompoundId
 
-    def __init__(
-            self, parentContainer, localId, referenceSetId=None):
+    def __init__(self, parentContainer, localId):
         super(AbstractFeatureSet, self).__init__(parentContainer, localId)
-        self._referenceSetId = referenceSetId
```

Comment:
Can you explain why you opted to attach the reference object to the classes as opposed to the identifier?